packages/react-ui-ag/src/Gmap/Markers.js

```diff
@@ -39,6 +39,8 @@ export default class Markers extends PureComponent {
     return feature => {
       const isBasic = get(feature, 'properties.isBasic', false)
       const scale = get(feature, 'properties.iconScale', 1)
+      const source = get(feature, 'properties.source', '')
+      const zIndex = typeof source === 'string' && source.toLowerCase() === 'aptguide' ? 2 : 1
```

Comment:
Is the `typeof` check really needed here? I thought `===` took care of that.

Reply:
`toLowerCase` is a string specific function, which is why I wanted to make sure it was a string first

Reply:
Would this be even better, perhaps?

```javascript
  typeof source.toLowerCase === 'function'
```

Reply:
If source is `undefined` that will cause a type error. "Cannot read property toLowerCase of undefined"